phoenixRest/models/core/event.py

```diff
@@ -69,32 +68,37 @@ class Event(Base):
 
     cancellation_reason = Column(Text)
 
-    def __init__(self, name: str, start_time: DateTime, end_time: DateTime, max_participants: int):
+    def __init__(self, name: str, start_time: DateTime, end_time: DateTime, booking_time: DateTime, priority_seating_time_delta: int, seating_time_delta: int, max_participants: int, participant_age_limit_inclusive: int, crew_age_limit_inclusive: int, theme: str, location_uuid: str, seatmap_uuid: str):
```

Comment:
I think the fields here should only be the fields that are not nullable. I.E why is theme provided?

Also you have a typing error. `theme` is nullable, yet the type is `str`. Use `Optional[str]` (importing it from `typing`).